src/main/main.cpp

```diff
@@ -329,6 +329,7 @@ catchup(Application::pointer app, uint32_t to, uint32_t count,
             break;
         }
         case LedgerManager::LM_CATCHING_UP_STATE:
+        case LedgerManager::LM_NUM_STATE:
```

Comment:
I typically prefer to put those like this, as it makes it obvious that it's unexpected
```
        case LedgerManager::LM_NUM_STATE: 
                abort();
```